modules/custom/az_global_footer/data/az_global_footer.json

```diff
@@ -44,17 +44,6 @@
       "enabled": true,
       "weight": "4"
     },
-    {
-      "link_id": "5",
-      "parent_link_id": "0",
-      "menu": "az-footer-main",
-      "title": "Annual Security Report",
-      "urlpath": "https://clery.arizona.edu/asr",
-      "external": true,
-      "expanded": false,
-      "enabled": true,
-      "weight": "5"
```

Comment:
Does the weight need to change in the other links that remain?

Reply:
yes